hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTask.java

```diff
@@ -112,6 +113,8 @@ public Pair<String, Boolean> process(OMUpdateEventBatch events) {
     if (!success) {
       LOG.error("processWithOBS failed.");
     }
+    LOG.debug("{} successfully processed in {} milliseconds",
```

Comment:
Is there a metric we publishes to Prometheus that covers this measurement?

Reply:
I think for Recon we do not.
Currently the metrics are only the ones we get are the same ones stored in RECON_TASK_STATUS table.
More implementation is in: [ReconTaskStatusMetrics](https://github.com/apache/ozone/blob/ad108c86b11606d49f8ea9b739e40340a2117fe2/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/metrics/ReconTaskStatusMetrics.java)

It will basically just fetch the values from DB and provide that as metrics.

Reply:
We can also figure out this metric by the difference in the `lastUpdatedTimestamp`.
For each task this metric is already present; read from DB. If we check the difference between the current and the previous timestamp we will get the time taken for a task to run.
But is good to have a debug level log as well.